<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QToolButton" name="ShapeButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Users/KOSA/Downloads/shape1.png</normaloff>../../../Users/KOSA/Downloads/shape1.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="ArrowButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Users/KOSA/Downloads/arrow.png</normaloff>../../../Users/KOSA/Downloads/arrow.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="MemoButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Users/KOSA/Downloads/memo.png</normaloff>../../../Users/KOSA/Downloads/memo.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="StraightMeasureButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Users/KOSA/Downloads/ruler.png</normaloff>../../../Users/KOSA/Downloads/ruler.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="TapeMeasureButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Users/KOSA/Downloads/tapemeasure.png</normaloff>../../../Users/KOSA/Downloads/tapemeasure.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="protractorButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Users/KOSA/Downloads/protractor.png</normaloff>../../../Users/KOSA/Downloads/protractor.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="toolButton_9">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="toolButton_8">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="toolButton_7">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>131</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QToolButton" name="toolButton_10">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="toolButton_11">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actiondraw_line"/>
   <addaction name="actiondraw_shape"/>
   <addaction name="separator"/>
  </widget>
  <action name="actiondraw_line">
   <property name="text">
    <string>draw_line</string>
   </property>
  </action>
  <action name="actiondraw_shape">
   <property name="text">
    <string>draw_shape</string>
   </property>
  </action>
  <action name="actionmemo">
   <property name="text">
    <string>memo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
